src/components/theme/View/ListingView.jsx

```diff
@@ -28,11 +28,13 @@ const ListingView = ({ content }) => (
             {item.description && <p>{item.description}</p>}
           </Container>
           {item.image_field && (
-            <PreviewImage
+            <Component
+              componentName="PreviewImage"
               item={item}
-              size="thumb"
-              alt={item.image_caption ? item.image_caption : item.title}
+              alt={item.image_caption}
```

Comment:
I am testing this and I found out we don't really receive the image_caption from restapi :S
And the reason why I was testing it is that I realized that if we were to receive caption data from restapi in listing blocks, we would not be considering we could have a preview image instead of the lead image. This means that just using image_caption, given that we receive it, could mean using the wrong caption.
I would suggest we just remove this alt from here and go with the empty one here, since it's not a regression to do so.

I tested creating a page in demo.plone.org with a listing block which shows a news item and an event.